src/desktop/config.coffee

```diff
@@ -66,11 +66,6 @@ module.exports =
   GEOIP_ENDPOINT: 'https://artsy-geoip.herokuapp.com/'
   GOOGLE_ADWORDS_ID: null
   GOOGLE_MAPS_API_KEY: null
-  INTERCOM_SELLER_APP_ID: null
-  INTERCOM_SELLER_ENABLED: false
-  INTERCOM_BUYER_APP_ID: null
-  INTERCOM_BUYER_APP_SECRET: null
-  INTERCOM_BUYER_ENABLED: false
```

Comment:
👍  We should also clean up the environment variables for staging and production.